Assets/Scripts/Potions/Potion.cs

```diff
@@ -10,6 +10,7 @@ public class Potion
     private int potionCoinValue;
     private Sprite potionImage;
     private List<CollectablePlantsEnum> plantsInPotion;
+    public string keyName = "55";
```

Comment:
I see where the 55 comes from now... wondering if there is something that's more intuitive we could add?